### ISSUE TITLE
Not able to install handsontable-pro thru npm.

### ISSUE THREAD

**Issue by bellcliff** [handsontable/handsontable-pro#16]
In our company we have a proxy setup for http/https, but on windows, the proxy does not work with the git protocol, which cause below error:

```bash
$ npm i -D handsontable-pro
npm ERR! Error while executing:
npm ERR! C:\Users\BYANG183\AppData\Local\Programs\Git\mingw64\bin\git.EXE ls-remote -h -t ssh://git@github.com/handsontable/formula.js.git
npm ERR!
npm ERR! ssh_exchange_identification: Connection closed by remote host
npm ERR! fatal: Could not read from remote repository.
npm ERR!
npm ERR! Please make sure you have the correct access rights
npm ERR! and the repository exists.
npm ERR!
npm ERR! exited with error code: 128
```

Any suggestion or walk around for this?

I did try install it from dev/mac machine, it all works, just it do not work with our windows machine, from where most of my collegues are working on.

I did try git clone https on our windows machine, it also works.

**Comment by AMBudnik** [comment]
Thank you for sharing.

**Comment by osecki** [comment]
Do you have a suggestion of how we can work around this issue? Is there another avenue (besides github issue) that we should use to get pro level tech support for this? Thanks.

**Comment by AMBudnik** [comment]
Hi @osecki 

we offer development consultancy as a commercial offer. For users that have Extended Support plan, it can be bought via support hours.

**Comment by Rex90** [comment]
Hi - I also have this issue through NPM. is there a known bug?

**Comment by AMBudnik** [comment]
The Handsontable PRO uses formulaJS library to handle Formula Support plugin. This library is loaded from Github, while it should be loaded from npm.
Here's a topic to follow for updates https://github.com/handsontable/handsontable-pro/issues/34

**Comment by swistach** [comment]
Hi @bellcliff, @osecki and @Rex90,
This issue is resolved in `hot-formula-parser@2.3.3`.
Please try to use `handsontable-pro@1.16.0`.

**Comment by swistach** [comment]
We can close this issue. Feel free to reopen this issue if `hot-formula-parser` still doesn't work in your use case.
